collect_app/src/main/res/xml/form_management_preferences.xml

```diff
@@ -14,7 +14,8 @@
             android:entries="@array/form_update_mode_entries"
             android:entryValues="@array/form_update_mode_values"
             android:key="form_update_mode"
-            android:title="@string/form_update_mode_title" />
+            android:title="@string/form_update_mode_title"
+            app:iconSpaceReserved="false" />
```

Comment:
Oops, I already merged Saumia's change for this. Probably should have assigned it to you in the first place.

Reply:
No worries!